docs/howto/pipeline_testing.md

```diff
@@ -81,30 +102,12 @@ The pipeline test runner transforms log lines into file input events and sends t
 }
 ```
 
-The `multiline` section configures the log file reader to correctly detect multiline log entries using the `first_line_pattern`. Use this property if your logs may be split into multiple lines, e.g. Java stack traces.
+The `multiline` section ([raw files](https://github.com/elastic/elastic-package/blob/master/docs/howto/pipeline_testing.md#raw-files) only) configures the log file reader to correctly detect multiline log entries using the `first_line_pattern`. Use this property if your logs may be split into multiple lines, e.g. Java stack traces.
```

Comment:
Nit: I think you can just use a relative link here (just `#raw-files`).

Reply:
Fixed, thanks!